src/components/YieldPools/ElasticFarmGroup/index.tsx

```diff
@@ -344,9 +341,11 @@ const ProMMFarmGroup: React.FC<Props> = ({ address, onOpenModal, pools, userInfo
         ) : (
           <Flex sx={{ gap: '12px' }} alignItems="center">
             {!account ? <ConnectWalletButton onClick={toggleWalletModal} /> : renderApproveButton()}
-            {account && canWithdraw && isApprovedForAll && (
+            {/*
+              account && canWithdraw && isApprovedForAll && (
               <ForceWithdrawButton onClick={() => onOpenModal('forcedWithdraw')} />
-            )}
+            )
+              */}
```

Comment:
remove?

Reply:
keep it for the future, in case contract have issue, we need to enable it again